android/2_build_toolchain.sh

```diff
@@ -76,7 +76,7 @@ function build() {
 	export PATH=$PLATFORM_PREFIX/bin:$PATH
 
 	export CFLAGS="-g0 -O2 $5"
-	export CXXFLAGS=$CFLAGS
+	export CXXFLAGS="$CFLAGS -DHB_NO_MMAP"
```

Comment:
We should only add such flags for libraries needing them → harfbuzz, libopus SIMD (#61).
But this will complicate this PR, so ignoring for now.

Reply:
Yeah this is for harfbuzz only. I wanted to pass CXXFLAGS= as part of install lib argument but didn't work.